sencha-workspace/SlateTasksTeacher/app/store/Students.js

```diff
@@ -0,0 +1,21 @@
+Ext.define('SlateTasksTeacher.store.Students',{
+    extend: 'Slate.cbl.store.Students',
+    requires: [
+        'Emergence.ext.proxy.Records'
+    ],
+    proxy: {
```

Comment:
@nbey the proxy config should be down below `config`, not up with the `Ext.define` options